### PR TITLE
ECO-982 More info admin

### PR DESCRIPTION
**Pull request by doodyparizada**
Add userWallet when viewing order history
Add all order attributes (all contexts) when viewing orders (instead of leaving some items blank)

### CODE REVIEW

File: scripts/src/models/orders.ts [hunk 1]
```diff
@@ -214,7 +214,7 @@ export const Order = {
 		const query = OrderImpl.createQueryBuilder("ordr")
 			.innerJoinAndSelect("ordr.contexts", "context")
 			.leftJoinAndSelect("context.user", "user")
-			.orderBy("ordr.current_status_date", "DESC")
+			.orderBy("ordr.currentStatusDate", "DESC")
```
Comment (doodyparizada) [inline scripts/src/models/orders.ts]: This is weird. It didn't kept failing without changing this but this code is in production too

**Comment by doodyparizada** [comment]
tested on beta

File: scripts/src/admin/services.ts [hunk 2]
```diff
@@ -487,23 +496,19 @@ export async function getApplicationUserData(params: { app_user_id: string, app_
...
-	}
 	if (query.user_id) {
-		const contexts = await OrderContext.find({ userId: query.user_id });
-		if (contexts.length > 0) {
-			q.andWhere("id in (:ids)", { ids: contexts.map(c => c.orderId) });
-		} else {
-			q.andWhere("id = 'no-such-id' ");
+		const userOrders = await Order.genericGet({ userId: query.user_id }).getMany();
+		let orderIds = userOrders.map(o => o.id);
+		if (!orderIds.length) {
```
Comment (liorama) [inline scripts/src/admin/services.ts]: 👍

File: scripts/src/admin/services.ts [hunk 3]
```diff
@@ -487,23 +496,19 @@ export async function getApplicationUserData(params: { app_user_id: string, app_
...
+			orderIds = ["no_orders"];
 		}
+		q.where(`ordr.id IN (:ids)`, { ids: orderIds });
+
 	}
-	const orders = await q.orderBy("current_status_date", "DESC").skip(skip(query)).take(take(query)).getMany();
+
+	const orders = await q.skip(skip(query)).take(take(query)).getMany();
```
Comment (liorama) [inline scripts/src/admin/services.ts]: Is the skip and take for pagination?

**Review by liorama — DISMISSED** [review]

File: scripts/src/admin/services.ts (hunk 3 above)
Comment (doodyparizada) [reply scripts/src/admin/services.ts]: yes

File: scripts/src/admin/services.ts (hunk 2 above)
Comment (doodyparizada) [reply scripts/src/admin/services.ts]: learnt from the best

**Pull request merged**